applications/StatisticsApplication/README.md

```diff
@@ -17,6 +17,140 @@ Following table summarize capabilities of Statistics Application.
+                            "method_settings": {}
+                        },
+                        {
+                            "method_name"    : "mean",
+                            "norm_type"      : "pnorm_2.5",
+                            "container"      : "nodal_non_historical",
+                            "variable_names" : ["VELOCITY", "LOAD_MESHES", "GREEN_LAGRANGE_STRAIN_TENSOR"],
+                            "method_settings": {}
+                        },
+                        {
+                            "method_name"    : "variance",
+                            "norm_type"      : "component_x",
+                            "container"      : "condition_non_historical",
+                            "variable_names" : ["VELOCITY"],
+                            "method_settings": {}
+                        },
+                        {
+                            "method_name"    : "rootmeansquare",
+                            "norm_type"      : "index_3",
+                            "container"      : "nodal_non_historical",
+                            "variable_names" : ["LOAD_MESHES"],
+                            "method_settings": {}
+                        },
+                        {
+                            "method_name"    : "min",
+                            "norm_type"      : "frobenius",
+                            "container"      : "nodal_non_historical",
+                            "variable_names" : ["GREEN_LAGRANGE_STRAIN_TENSOR"],
+                            "method_settings": {}
+                        }
+                    ],
+                    "output_settings" : {
+                        "output_control_variable": "STEP",
+                        "output_time_interval"   : 1,
+                        "write_kratos_version"   : false,
+                        "write_time_stamp"       : false,
+                        "output_file_settings"   : {
+                            "file_name"  : "<model_part_name>_<container>_<norm_type>_<method_name>.dat",
+                            "folder_name": "spatial_statistics_process",
+                            "write_buffer_size" : -1
+                        }
+                    }
+                }
+            }
+```
+
+### Temporal statistics process example
+
+Following is an example on how to use `temporal_statistics_process` in json. The output variables data type should be matched with the input variables order and the data type for `"norm_type" = "none"`. If `"norm_type" != "none"`, then output variables should be `scalars`. If `"container" = "nodal_historical_historical"` is used as the container type, then output variables should be added to `NodalSolutionStepVariables` list in Kratos since this container type outputs temporal statistics variables to nodal historical container. This json settings also can be added to `auxiliary_processe` list.
```

Comment:
```suggestion
Following is an example on how to use `temporal_statistics_process` in json. The output variables data type should be matched with the input variables order and the data type for `"norm_type" = "none"`. If `"norm_type" != "none"`, then output variables should be `scalars`. If `"container" = "nodal_historical_historical"` is used as the container type, then output variables should be added to `NodalSolutionStepVariables` list in Kratos since this container type outputs temporal statistics variables to nodal historical container. This json settings also can be added to `auxiliary_processes` list.
```